deps/build.jl

```diff
@@ -49,46 +49,26 @@ const dlprefix = Compat.Sys.iswindows() ? "" : "lib"
 
 # return libpython name, libpython pointer
 function find_libpython(python::AbstractString)
-    # it is ridiculous that it is this hard to find the name of libpython
-    v = pyconfigvar(python,"VERSION","")
-    libs = [ dlprefix*"python"*v, dlprefix*"python" ]
-    lib = pyconfigvar(python, "LIBRARY")
-    lib != "None" && pushfirst!(libs, splitext(lib)[1])
-    lib = pyconfigvar(python, "LDLIBRARY")
-    lib != "None" && pushfirst!(pushfirst!(libs, basename(lib)), lib)
-    libs = unique(libs)
-
-    # it is ridiculous that it is this hard to find the path of libpython
-    libpaths = [pyconfigvar(python, "LIBDIR"),
-                (Compat.Sys.iswindows() ? dirname(pysys(python, "executable")) : joinpath(dirname(dirname(pysys(python, "executable"))), "lib"))]
-    if Compat.Sys.isapple()
-        push!(libpaths, pyconfigvar(python, "PYTHONFRAMEWORKPREFIX"))
+    cmd = `$python $(joinpath(@__DIR__, "find_libpython.py"))`
+    if "yes" == get(ENV, "PYCALL_DEBUG_BUILD", "no")
+        cmd = `$cmd --verbose`
+    end
+    if VERSION < v"0.7-"
+        pipe, process = open(cmd)
+    else
+        pipe = process = open(cmd)
+    end
+    libpy_name = read(pipe.out, String)
```

Comment:
ATM `find_libpython.py` is run in the default "print the best one" mode. I can also run it with `--list-all` option, loop over all the candidate paths and use the first one that `Libdl.dlopen` succeeds. What do you think?